evergreen.yml

```diff
@@ -921,39 +974,53 @@ tasks:
       - func: "install rust toolchain"
       - func: "run windows result set test"
 
+  - name: windows-sign
```

Comment:
I think that the release task on Windows should depend on `windows-sign`

Reply:
Oh yea, this is a bad oversight. Surprised all my patches worked, definitely just by luck.

Reply:
That's because the release task does not run on PR tests 😄 Only tags will trigger it.

Reply:
I was running it manually ;)